app/src/renderer/components/common/TmSessionSignUp.vue

```diff
@@ -226,14 +226,14 @@ export default {
           account: this.fields.signUpName,
           optin: this.fields.errorCollection
         })
-        this.$store.commit(`setModalSession`, false)
+        this.$store.commit(`toggleSignInModal`, false)
```

Comment:
maybe the parent component should stay as `sessionModal` since sign up is included.